transport-protocol/src/custom.rs

```diff
@@ -309,6 +342,15 @@ pub fn serialize_sgx_attestation_init(pubkey: &[u8], device_id: i32) -> Transpor
     Ok(transport_protocol.write_to_bytes()?)
 }
 
+pub fn serialize_cert_chain(enclave_cert: &[u8], root_cert: &[u8]) -> TransportProtocolResult {
```

Comment:
Both `enclave_cert` and `root_cert` are better as owned values rather than borrows, I think.